README.md

```diff
@@ -272,28 +264,12 @@ See gif in action:
 
 ### Invoking VSCode actions from neovim
 
-There are a
-[few helper functions](https://github.com/asvetliakov/vscode-neovim/blob/master/vim/vscode-neovim.vim#L17-L39) that are
-used to invoke VSCode commands from Neovim:
-
-| Command                                                                                                                                                           | Description                                                                                                                                                                                                |
-| ----------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `VSCodeNotify(command, ...)` <br/> `VSCodeCall(command, ...)`                                                                                                     | Invoke VSCode command with optional arguments.                                                                                                                                                             |
-| `VSCodeNotifyRange(command, line1, line2, leaveSelection ,...)` <br/> `VSCodeCallRange(command, line1, line2, leaveSelection, ...)`                               | Produce linewise VSCode selection from `line1` to `line2` and invoke VSCode command. Setting `leaveSelection` to 1 keeps VSCode selection active after invoking the command.                               |
-| `VSCodeNotifyRangePos(command, line1, line2, pos1, pos2, leaveSelection ,...)` <br/> `VSCodeCallRangePos(command, line1, line2, pos1, pos2, leaveSelection, ...)` | Produce characterwise VSCode selection from `line1.pos1` to `line2.pos2` and invoke VSCode command.                                                                                                        |
-| `VSCodeNotifyVisual(command, leaveSelection, ...)` <br/> `VSCodeCallVisual(command, leaveSelection, ...)`                                                         | Produce linewise (visual line) or characterwise (visual and visual block) selection from visual mode selection and invoke VSCode command. Behaves like `VSCodeNotify/Call` when visual mode is not active. |
-
-> 💡 Functions with `Notify` in their name are non-blocking, the ones with `Call` are blocking. Generally **use Notify**
-> unless you really need a blocking call.
+To invoke VSCode actions from Neovim, use `VSCodeNotify(command, args...)` or `VSCodeCall(command, args...)` functions.
```

Comment:
Is there a Lua interface also?

```suggestion
To invoke VSCode actions from Neovim, use:
- from Lua: `require('vscode').notify(command, args...)` or `require('vscode').call(command, args...)`
- from Vimscript: `VSCodeNotify(command, args...)` or `VSCodeCall(command, args...)`
```

Reply:
Not yet, though I hope to add one soon. Currently, just `vim.fn.VSCodeNotify(...)`.